packages/main/src/ColorPalettePopover.js

```diff
@@ -0,0 +1,190 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import CSSColor from "@ui5/webcomponents-base/dist/types/CSSColor.js";
+import { isPhone } from "@ui5/webcomponents-base/dist/Device.js";
+import ColorPalettePopoverTemplate from "./generated/templates/ColorPalettePopoverTemplate.lit.js";
+
+// Styles
+import ColorPalettePopoverCss from "./generated/themes/ColorPalettePopover.css.js";
+import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverCommon.css.js";
+
+import Popover from "./Popover.js";
+import Button from "./Button.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import ColorPalette from "./ColorPalette.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-color-palette-popover",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.ColorPalettePopover.prototype */ {
+		/**
+		 * Defines whether the user can see the last used colors in the bottom of the component
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 * @since 1.0.0-rc.19
```

Comment:
For this component, you should add the since tag to the class only, not to each property separately because the whole component is available after this version